Library/Formula/libu2f-server.rb

```diff
@@ -0,0 +1,26 @@
+class Libu2fServer < Formula
+  desc "Yubico Universal 2nd Factor (U2F) Server C Library"
+  homepage "https://developers.yubico.com/libu2f-server/"
+  url "https://developers.yubico.com/libu2f-server/Releases/libu2f-server-1.0.1.tar.xz"
+  sha256 "a618f59051209d6d70c24cf42d64c9b67bd7dd5946b6dbd2c649181d7e8f1f6e"
+
+  depends_on "check" => :build
+  depends_on "json-c"
+  depends_on "openssl"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    ENV["LIBSSL_LIBS"] = "-lssl -lcrypto -lz"
+    ENV["LIBCRYPTO_LIBS"] = "-lcrypto -lz"
+
+    ENV["HELP2MAN"] = "/usr/local/bin/help2man --no-discard-stderr"
+    ENV["PKG_CONFIG"] = "/usr/local/bin/pkg-config"
+
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/u2f-server", "--version"
```

Comment:
See other note (https://github.com/Homebrew/homebrew/pull/43676#discussion_r38879487).

pam_yubico also does something similar, since it requires a USB key to be inserted: https://github.com/Homebrew/homebrew/blob/1777fc04b5afbc1ed309361c1324cfe62ce32f62/Library/Formula/pam_yubico.rb